src/fuse_concat.cpp

```diff
@@ -50,8 +50,12 @@ struct fused_concat
     shape compute_shape(std::vector<shape> inputs, const std::vector<module_ref>& mods) const
     {
         check_shapes{inputs, *this}.same_ndims();
-        if((inputs.size() + 1) == mods.size())
-            MIGRAPHX_THROW("FUSED_CONCAT: Missing fused modules");
+        // original concat can have multiple inputs. Let's say it has `n` input args.
+        // Each of those `n` input args are converted into pointwise modules that take atleast 1
+        // input parameter. Fused concat will have `n+1` module arguments. `n+1`th module is the
+        // post pointwise module which can take 0 or more input arguments.
+        if((inputs.size() + 1) < mods.size())
```

Comment:
Your comment, does that imply that we should be checking <= instead of just < here?

Reply:
No, `==` is a valid. It needs to check if its' invalid and throw.

Reply:
added assert for the `input_iter`